internal/internal.go

```diff
@@ -860,7 +861,8 @@ func (i *Internal) GetTimeLimitHandler(c echo.Context) error {
 		return err
 	}
 
-	outputMap, err := i.getTimeLimit(userID, analysisID)
+	// err is not checked here -- should it be?
+	outputMap, err := i.getTimeLimit(userID, analysisID) // nolint
```

Comment:
As mentioned. Could throw in the usual `if err != nil { return err }` here if it should be checked.

Reply:
Huh, weird. Yeah, I'd recommend adding in the check.